src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -320,6 +322,10 @@ private ContentAccessCertificate updateScaCertificate(Consumer consumer, Owner o
         return existing;
     }
 
+    private void revoke(ContentAccessCertificate existing) {
```

Comment:
We probably want a separate PR for this, as this seems important enough for its own call out.

Reply:
Done.